src/main/resources/jupyter/init-actions.sh

```diff
@@ -60,6 +60,7 @@ if [[ "${ROLE}" == 'Master' ]]; then
     JUPYTER_CUSTOM_JS_URI=$(jupyterCustomJsUri)
     JUPYTER_GOOGLE_SIGN_IN_JS_URI=$(jupyterGoogleSignInJsUri)
     JUPYTER_USER_SCRIPT_URI=$(jupyterUserScriptUri)
+    JUPYTER_NOTEBOOK_CONFIG_URI=$(jupyterNotebookConfig)
```

Comment:
Minor, the pattern for other templated fields that are substituted with URIs are like: `$(jupyterNotebookConfigUri)